tests/acceptance/fixtures.py

```diff
@@ -118,7 +118,7 @@ def board_cleanup():
     request.addfinalizer(board_cleanup)
 
 
-def setup_qemu(request, build_dir, conn):
+def setup_qemu(request, qemu_wrapper, build_dir, conn):
```

Comment:
minor: I wonder why you added that as a second and not last argument?

Reply:
Because it is the most important, IMO. Just personal taste, I guess :)